src/frontend/app.py

```
@@ -0,0 +1,404 @@
+                )
+
+                st.json(data)
+
+            else:
+                st.error(
+                    f"Ошибка: {response.status_code}"
+                )
+
+                st.json(data)
+
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)